client/src/components/LogInModal.jsx

```diff
@@ -42,49 +38,77 @@ const LogInModal = ({ isOpen, onClose }) => {
                 }
             } catch (error) {
-                //Update this error handling as well once log in is complete
-                console.error({ message: "Error checking user name", details: error });
+                setFormErrorMessage("Please try again.");
+                console.error({ message: "Error checking credentials", details: error });
             }
         } else {
-            //Placeholder alert for temporary error handling - will implement robust form error handling while building out full log in component in Week 3
-            alert("Please make sure to enter a username.");
+            setFormErrorMessage("Please enter a username and password.");
         }
     }
 
+    //State to hold inputted information
+    const [credentials, setCredentials] = useState({});
+
+    //Update state holding inputs as the user types
+    const handleChange = (event) => {
+        //Pull name associated with question -> will translate to column name in database
+        const name = event.target.name;
 
+        //Pull value inputted to answer field to save for form submission
+        const value = event.target.value;
+
+        //Update credentials state to hold all inputted answers
+        setCredentials(prevCredentials => ({ ...prevCredentials, [name]: value }));
+    }
+
+    //Function to clear inputs and exit modal
+    const cancel = () => {
+        setCredentials({});
+        onClose();
+    }
     
     return (
         <div className="modal-overlay" onClick={onClose}>
             <div className="modal-content" onClick={(e) => e.stopPropagation()}>
                 <form onSubmit={handleSubmit}>
-                    <label htmlFor="username">Pick User</label><br></br>
-                    <select name="username">
-                        <option key="1" value="user1">Test User 1</option>
-                        <option key="2" value="user2">Test User 2</option>
-                        <option key="3" value="user3">Test User 3</option>
-                        <option key="4" value="user4">Test User 4</option>
-                        <option key="5" value="user5">Test User 5</option>        
-                    </select>
-
+                    <label htmlFor="email">Email</label><br></br>
+                    <input
+                        type="email"
+                        id="email"
+                        name="email"
+                        value={credentials.email || ""}
+                        onChange={handleChange}
+                    /><br></br><br></br>
+
+                    <label htmlFor="password">Password</label><br></br>
+                    <input
+                        type="password"
+                        id="password"
+                        name="password"
+                        value={credentials.password || ""}
+                        onChange={handleChange}
+                    /><br></br><br></br>
```

Comment:
If you find you are using a bunch of br tags, you can use <div> and styling to help create the expected spacing you are needing. This is a good example of a jr implementation compared to someone with more experience.

Reply:
Thank you for calling this out! I had forgotten about those. I've updated them within my styling branch which will be merged shortly.